src/components/item/MapView.tsx

```diff
@@ -9,18 +9,21 @@ import { buildPlayerTabName } from '@/config/selectors';
 
 type Props = {
   parentId?: DiscriminatedItem['id'];
-  title: string;
+  title?: string;
+  height?: string;
 };
 
-const MapView = ({ parentId, title }: Props): JSX.Element => {
+const MapView = ({ parentId, title, height = '80vh' }: Props): JSX.Element => {
```

Comment:
The container is a Flex item, so it should be possible to grow it to fill the page.